data.sql

```diff
@@ -9,3 +9,10 @@ VALUES ('Pikachu', date '2021-01-07', 1 , 'false', 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', date '2017-05-12', 5 , 'true', 11);
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', date '2020-02-08', 0, 'false', 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', date '2021-11-15', 2, 'true', 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', date '1993-04-02', 3, 'false', 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', date '2005-06-12', 1, 'true', 45);
```

Comment:
* Kindly fix these weights, according to the weight provided in the project requirement you have them in negative numbers 
![image](https://user-images.githubusercontent.com/52497246/206014068-a0c19878-c836-4526-80e5-4f26831b040c.png)